flipper/__init__.py

```diff
@@ -0,0 +1,55 @@
+import cv2
+
+
+class Flipper(object):
+    """Class to encapsulate all the flipper logic.
+
+    Intended flow is:
+
+        left_flipper = Flipper('left')
```

Comment:
Why do you need a name? Seems like variable name that you assign this instance to will be the name you're looking for.